src/components/ThymeTable/Entry.js

```diff
@@ -153,22 +161,37 @@ class Entry extends Component<EntryType, EntryStateType> {
 
   onStartTimeTracking = () => {
     const { entry } = this.state;
-
-    const startTime = new Date();
-
+    let startTime = isEqual(entry.start, defaultStart)
```

Comment:
I will think a little about the code in these two methods. I feel like we could do some smart stuff using https://date-fns.org/docs/Getting-Started